onprc_ehr/resources/etls/HansenQueriesToFile.xml

```diff
@@ -12,110 +13,80 @@
         <transform id="step2">
             <description>Export Query to File Share</description>
             <source queryName="RandData_AnimalGroups" schemaName="onprc_ehr"/>
-            <destination type="file" dir="hansenFiles" fileBaseName="AnimalGroups-${TransformRunId}-${Timestamp}"
```

Comment:
The lack of a <destination> means that you won't be able to run this ETL. That's fine from the automated test perspective, but will of course be an issue in the real-world.